scripts/holochain-client.js

```diff
@@ -8,7 +8,6 @@ async function initAndGetHolochainClient () {
   if (holochainClient) return holochainClient
   try {
     holochainClient = await hcWebClientConnect({
-      url: process.env.REACT_APP_DNA_INTERFACE_URL,
```

Comment:
why remove this env var?

Reply:
After preparing our build for holoscape and building around recommended hc patterns, I highly recommend referencing logic that determines whether we're in a development or production environment and assigns the url value accordingly.  So, this is a cleanup task.  I will turn this into a logical statement. :+1: